spring-cloud-commons/src/test/java/org/springframework/cloud/client/loadbalancer/reactive/ReactorLoadBalancerExchangeFilterFunctionTests.java

```diff
@@ -164,12 +164,12 @@ void loadBalancerLifecycleCallbacksExecuted() {
 	@RestController
 	static class Config {
 
-		@RequestMapping("/hello")
+		@GetMapping("/hello")
```

Comment:
In a normal app, the `@RequestMapping` that this replaces represents a CSRF-injection vulnerability.